pgmq-rs/tests/pg_ext_integration_test.rs

```diff
@@ -157,7 +157,7 @@ async fn test_ext_send_read_delete() {
         .expect("failed to delete");
     assert!(deleted);
 
-    // try to delete a message that doesnt exist
+    // try to delete a message that doesn't exist
```

Comment:
this is triggering CI to fail on pgmq-rs. only because its telling us that the version in pgmq-rs/Cargo.toml already exists in crates.io. We can ignore that CI error.